GRID/utils/grid_submit.sh

```diff
@@ -575,7 +576,7 @@ export PATH=$PATH:$PWD
 # ----------- EXECUTE ACTUAL JOB  ------------------------------------ 
 # source the actual job script from the work dir
 chmod +x ./alien_jobscript.sh
-./alien_jobscript.sh
+./alien_jobscript.sh ${SCRIPTARGS}
```

Comment:
This is unfortunately not as easy as this. One would, in addition, need to pass around the arguments in the JDL specification. Honestly, I would avoid doing this change for now. You can easily write a script which creates jobfiles containing the right arguments before submitting.

Reply:
Ok, I reverted the changes in the grid_submit script.